src/tools/ceph-dencoder/osd_types.h

```diff
@@ -24,12 +25,14 @@ TYPE(pg_hit_set_history_t)
 TYPE(pg_history_t)
 TYPE(pg_info_t)
 TYPE(PastIntervals)
+TYPE(PastIntervals::pg_interval_t)
+using pg_interval_t = PastIntervals::pg_interval_t;
+TYPE(pg_interval_t)
```

Comment:
Why do we need both the namespace alias and the original namespaces declared?
I noticed that this was the case also with `ceph::os::Transaction`, `MgrMap::StandbyInfo`, etc.

Reply:
ceph-dencoder doesn't know how to find type when it doesn't have its namespace, and since we dumped the type without namespace we will "lose" checks for older version, but yes, i'll change that to skip those types, but will add dump for the full namespace